### PR TITLE
Create contact me page

### PR DESCRIPTION
**Pull request by SantiM1**
Create a contact me  page by using servlets and forms, also adding an acknowledgement page after the submission.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        ArrayList<String> personalInfo = new ArrayList<String>();
+        personalInfo.add(currentInfo);
+        personalInfo.add(pastInfo);
+        personalInfo.add(futureInfo);
+        String json = convertToJson(personalInfo);
+        //Display
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+    private String convertToJson(ArrayList<String> info) {
```
Comment (ChenMeng-SPS22) [inline portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java:33]: This is the "stupid" way to do it as the example wants to show you in comparison to gson.convertToJson. You should use gson one instead.

File: portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        //Strings
+        String currentInfo = "I'm taking finals this week";
+        String pastInfo = "I did a few art commisions this past month" ;
+        String futureInfo = "I'll be aplying to community service programs this summer";
+        //Create array of strings
+        ArrayList<String> personalInfo = new ArrayList<String>();
+        personalInfo.add(currentInfo);
+        personalInfo.add(pastInfo);
+        personalInfo.add(futureInfo);
+        String json = convertToJson(personalInfo);
```
Comment (ChenMeng-SPS22) [inline portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java]: I think the correct parameter to gson.convertToJson is a Java Object(From a java class). A Json object has a "tag" and a "body", while an ArrayList element only has a "body" so they cannot be directly converted.
See **software-product-sprint/walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/data/ServerStats.java** for a Java class that holds the data.

**Review by ChenMeng-SPS22 — CHANGES_REQUESTED** [review]
Hi Santiago, great progress! Please see my comment for how to use gson correctly, a great opportunity to learn :)

File: portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java (hunk 1 above)
Comment (SantiM1) [reply portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java:33]: Yeah, I did try usign gson but I couldn't get it to work, it was sending the request correctly but it didn't appear on my page for some reason. I'll try to get it to cooperate again and see if it was something else I had changed later.

File: portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java (hunk 2 above)
Comment (SantiM1) [reply portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java]: Alright, I'll check this out.

Comment (SantiM1) [reply portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java]: Worked like a charm, thanks Chen

File: portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java (hunk 1 above)
Comment (ChenMeng-SPS22) [reply portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java:33]: I think you are using it correctly right now, if it works you can remove the commented code. Let me know if you have any question regarding gson.

**Review by ChenMeng-SPS22 — APPROVED** [review]
Great work Santiago!

**Pull request merged**